### PR TITLE
Enable Ansible Lightspeed

### PR DESCRIPTION
**Pull request by svor**
Add .code-workspace file with settings to enable Ansible Lightspeed automatically.

screenshot-nimbusweb me-2024 07 23-12_51_21

**Related issue:** https://issues.redhat.com/browse/CRW-6336

**How to test:**
1/ Create a workspace using
[Contribute](https://devspaces.apps.sandbox-stage.gb17.p1.openshiftapps.com/dashboard/#/https://github.com/devspaces-samples/ansible-devspaces-demo/tree/sv-enable-lightspeed)
2/ When the workspace is started and Ansible extension is initialized, Ansible Lightspeed should be enabled and it should be ready to connect to the server 

It's not possible to do this via .vscode/settings.json file because Ansible extension doesn't apply `ansible.lightspeed.enabled` settings that was set on Workspace Folder level. Here is the corresponding issue: https://github.com/ansible/vscode-ansible/issues/1443

### CODE REVIEW

**Comment by svor** [comment]
@ibuziuk @dmytro-ndp could you please take a look

**Comment by dmytro-ndp** [comment]
@svor : I have tested PR on https://devspaces.apps.sandbox-stage.gb17.p1.openshiftapps.com/ using factory URL https://github.com/devspaces-samples/ansible-devspaces-demo/tree/sv-enable-lightspeed

I can confirm that I was able to connect to "Ansible Lightspeed" service without the problem without additional modification of VS Code Editor settings, even though it was not enabled in the `Settings > Ansible extenstion > Ansible Lightspeed`
Screenshot from 2024-07-23 15-32-39

And the same result was on workspaces.openshift.com:
ansible-workspaces openshift com

Good job, @svor !

**Review by dmytro-ndp — APPROVED** [review]
looks good to merge

File: .code-workspace [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"settings": {
+		"ansible.lightspeed.suggestions.enabled": true,
+		"ansible.lightspeed.enabled": true
+	},
+	"folders": [
```
Comment (vitaliy-guliy) [inline .code-workspace:6]: What if we remove folders at all?
Just not to hardcode `/projects` here.

Comment (svor) [reply .code-workspace:6]: `folders` block will be added automatically when workspace is created and we'll have unstashed changes in the workspace

**Pull request merged**